controlplane/src/core/repositories/analytics/MetricsDashboardRepository.ts

```diff
@@ -321,7 +321,7 @@ export class MetricsDashboardRepository {
         top:
           top5Response.data?.result.map((v: any) => ({
             name: v.metric.wg_operation_name || 'unknown',
-            value: v.value[1] || '0',
+            value: !v.value[1] || v.value[1] === 'NaN' ? '0' : v.value[1],
```

Comment:
Maybe other places would face the same issue? I saw it with P95 latency too. (the big number in the card) We should have a resolveValue fucntion and apply it to every field here where we access value

Reply:
Good point